EVSL_1.0/TESTS/GEN/Lan_MMPLanR.c

```diff
@@ -123,38 +118,19 @@ int main() {
         if (Bcoo.nrows != n) {
           return 1;
         }
-        // nnz = Bcoo.nnz;
-        // n = Bcoo.nrows;
-        // printf("size of B is %d\n", n);
-        // printf("nnz of  B is %d\n", nnz);
       } else {
         fprintf(flog, "read_coo error for B = %d\n", ierr);
         exit(6);
       }
-      /*------------------ diagonal scaling for Acoo and Bcoo */
-      sqrtdiag = (double *)calloc(n, sizeof(double));
-      extractDiag(&Bcoo, sqrtdiag);
-      diagScaling(&Acoo, &Bcoo, sqrtdiag);
```

Comment:
If we want to use the Choleksy factor of B for DOS, we do not need diagonal scaling. Otherwise, we still have to use diagonal scaling.